src/components/Table/common/utils.ts

```diff
@@ -304,24 +266,6 @@ export function getPinnedCellIndex<T extends RowData>(
   return [visibleCells[pinnedIndex], pinnedIndex];
 }
 
-/**
- * Returns table sort label props.
- * @param column - Table column.
- * @returns table sort label props.
- */
-export function getTableSortLabelProps<T extends RowData>(
```

Comment:
`getTableSortLabelProps` is deleted and the props are added directly to the `TableSortLabel` component in `TableHead`.